src/hc-06.hpp

```diff
@@ -0,0 +1,99 @@
+// ==========================================================================
```

Comment:
Incorrect use of Doxygen, please read the wiki for the correct Doxygen standard.